src/main/java/movie/storage/util/HibernateUtil.java

```diff
@@ -0,0 +1,24 @@
+package movie.storage.util;
+
+import movie.storage.exception.IncorrectDataException;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new IncorrectDataException("Failed to build SessionFactory", e);
```

Comment:
let's leave just runtime exception here, its different type of exception than in Dao